--- a/profiles/openberkeley/openberkeley_tests/behat/features/panopoly/livepreview.feature
+++ b/profiles/openberkeley/openberkeley_tests/behat/features/panopoly/livepreview.feature
@@ -20,9 +20,6 @@ Feature: Live preview
     # We need to check the table header case insensitively, because it's not
     # uncommon to make table headers capitalized.
     Then I should see text matching "/c-1-r-1/i" in the "Live preview" region
-    When I fill in "tablefield_0_cell_0_1" with "c-2-r-1"
-      And I wait for live preview to finish
-    Then I should see text matching "/c-2-r-1/i" in the "Live preview" region
     # Test that we can make the title into a link
     Then I should not see the link "Widget title" in the "Live preview" region
     When I check the box "Make title a link"
@@ -36,7 +33,6 @@ Feature: Live preview
     Then I should see "Widget title"
       And I should see the link "Widget title"
       And I should see "c-1-r-1"
-      And I should see "c-2-r-1"

   @api @javascript @panopoly_magic @panopoly_widgets
   Scenario: Live preview should work with views
